site/weave-docker-api.md

```diff
@@ -10,6 +10,21 @@ network when they are started using the ordinary Docker
 or the [remote API](https://docs.docker.com/reference/api/docker_remote_api/),
 instead of `weave run`.
 
+###Attaching Containers to a Weave Network
+
+There are two ways to attach containers to a Weave network, 
+the first uses the Weave Net Docker API Proxy, and is described in this section.  
+The other method uses the Docker Network Plugin framework. It’s your choice which method to use. 
+
+To use the Docker Network Plugin, launch Weave with the --net flag.  Note that after 
+you’ve launched Weave Net with this flag, then the Weave Docker API Proxy is automatically disabled.  
```

Comment:
> Finally, there is another option that has the same effect: -e WEAVE_CIDR=none. It's mentioned in a few places in the docs. I reckon that option and the explanation about --net should be consolidated into one place.

Should we put this together with the --net flag explanation in their own place or should I add the -e WEAVE_CIDR=none option here to the --net flag point as well.  

It seems to me that this whole section could be put higher up in the docs or at least a link to it, so that people can read about this right away.  thoughts?